torch/csrc/profiler/standalone/nvtx_observer.cpp

```diff
@@ -95,7 +94,7 @@ static std::list<std::pair<at::RecordFunctionHandle, int>> getInputTensorOpIds(
     } else {
       if (input_item.isList()) {
         std::list<std::pair<at::RecordFunctionHandle, int>> tmp_op_ids =
-            flattenOpIdList(input_item.toList(), std::string(fn.name()));
```

Comment:
Why was this arg removed? Is this intentional?

Reply:
It is intentional since flattenOpIdList is a static function and the second parameter is not used.